code/src/App.css

```diff
@@ -0,0 +1,108 @@
+header {
+  display: flex;
+}
+
+.header-title {
+  font-size: 29px;
+  margin: auto;
+  padding-top: 20px;
+  color: #ECCCCF;
+  opacity: 80%;
```

Comment:
Its kind of hard to se the title, since the color contrasts are very small, so consider maybe adding a background with less opacity or a backdropfilter to the h1 then it will bigger contrast to the background, or change the text-color to a darker color!